packages/db-base/src/lib/redisHandler.ts

```diff
@@ -366,9 +424,9 @@ class RedisHandler extends EventEmitter {
     /**
      * Encode an Array depending on the type of the elements
      * @param arr Array to encode
-     * @returns {Array} Array with Buffers with encoded values
+     * @returns Array with Buffers with encoded values
      */
-    encodeRespArray(arr) {
+    encodeRespArray(arr: any[]): Buffer[] {
```

Comment:
`unknown[]` would probably be the safer choice since all possible variants need to be handled somehow.

Reply:
Same goes for the methods that call this.